sql/mob_droplist.sql

```diff
@@ -33027,6 +33027,37 @@ INSERT INTO `mob_droplist` VALUES (4658,0,0,1000,2814,250); -- (Big Bang) Aptant
 INSERT INTO `mob_droplist` VALUES (4659,0,0,1000,11496,48); -- (Lizardtrap) Fenrir's Crown
 INSERT INTO `mob_droplist` VALUES (4660,0,0,1000,16035,41); -- (Goaftrap) Altdorf's Earring
 
+INSERT INTO `mob_droplist` VALUES (4661,0,0,1000,1628,68); -- (King Buffalo) Buffalo Hide
+INSERT INTO `mob_droplist` VALUES (4661,0,0,1000,5152,130); -- (King Buffalo) Buffalo Meat
+INSERT INTO `mob_droplist` VALUES (4661,0,0,1000,1615,25); -- (King Buffalo) Buffalo Horn
+INSERT INTO `mob_droplist` VALUES (4661,0,0,1000,5703,285); -- (King Buffalo) Uleguerand Milk
+INSERT INTO `mob_droplist` VALUES (4662,0,0,1000,928,1000); -- (Will-O'-The_Wykes) Pinch of bomb ash
+INSERT INTO `mob_droplist` VALUES (4662,0,0,1000,17316,200); -- (Will-O'-The_Wykes) Bomb Arm
+INSERT INTO `mob_droplist` VALUES (4663,0,0,1000,511,0); -- (Goblin Jeweler) Goblin mask
+INSERT INTO `mob_droplist` VALUES (4663,0,0,1000,4518,0); -- (Goblin Jeweler) Strip of sheepjerky
+INSERT INTO `mob_droplist` VALUES (4663,0,0,1000,4509,0); -- (Goblin Jeweler) Flask of distilled  water
+INSERT INTO `mob_droplist` VALUES (4663,0,0,1000,605,30); -- (Goblin Jeweler) Pickaxe
```

Comment:
http://ffxidb.com/zones/9/goblin-jeweler
None of Gob Jewlers drops look legit. Suggest asking someone on retail to get some screencaps confirming that these even drop with TH zero, and then we can try and estimate a base rate since there isn't enough data yet.